__docs__/wonder-blocks-tokens/__overview__.mdx

```diff
@@ -20,10 +20,11 @@ building blocks of our design system.
 
 These represent the design decisions at Khan Academy, such as:
 
--   <a href="../?path=/docs/packages-tokens-border--docs">Border</a>
--   <a href="../?path=/docs/packages-tokens-color--docs">Color</a>
--   <a href="../?path=/docs/packages-tokens-spacing--docs">Spacing</a>
--   <a href="../?path=/docs/packages-tokens-typography--docs">Typography</a>
+-   <a href="/?path=/docs/packages-tokens-border--docs">Border</a>
+-   <a href="/?path=/docs/packages-tokens-semantic-color--docs">Semantic Colors</a>
+-   <a href="/?path=/docs/packages-tokens-color--docs">Color Primitives</a>
+-   <a href="/?path=/docs/packages-tokens-spacing--docs">Spacing</a>
+-   <a href="/?path=/docs/packages-tokens-typography--docs">Typography</a>
```

Comment:
**note:** Reworked this section to include semantic tokens + differentiate them from the primitive ones.

/?path=/docs/packages-tokens-overview--docs

<img width="1069" alt="Screenshot 2024-08-15 at 5 44 17 PM" src="https://github.com/user-attachments/assets/b955afac-6d8d-4277-88aa-62c054dd6651">